integrations/clickup/clickup/client.py

```diff
@@ -0,0 +1,186 @@
+        """Get all spaces in a workspace."""
+        url = f"{self.api_url}/team/{team_id}/space"
+        params = {"archived": "false"}
+        response = await self._send_api_request(url, params)
+        spaces = response.get("spaces")
+        for space in spaces:
+            yield space.get("id")
+
+    async def _get_folders_in_space(self, team_id: str) -> AsyncGenerator[str, None]:
+        """Get all folders in a space."""
+        async for space in self._get_spaces_in_team(team_id):
+            url = f"{self.api_url}/space/{space}/folder"
+            params = {"archived": "false"}
+            response = await self._send_api_request(url, params)
+            folders = response.get("folders")
+            for folder in folders:
+                yield folder.get("id")
+
+    async def get_folder_projects(self) -> AsyncGenerator[List[dict[str, Any]], None]:
+        """Get all projects with a folder parent."""
+        for team_id in self.team_key:
+            async for folder_id in self._get_folders_in_space(team_id):
+                url = f"{self.api_url}/folder/{folder_id}/list"
+                params = {"archived": "false"}
+                response = await self._send_api_request(url, params)
+                projects = response.get("lists")
+                yield [
+                    {**project, CustomProperties.TEAM_ID: team_id}
+                    for project in projects
+                ]
+
+    async def get_folderless_projects(
+        self,
+    ) -> AsyncGenerator[List[dict[str, Any]], None]:
+        """Get all projects without a folder parent."""
+        for team_id in self.team_key:
+            async for space_id in self._get_spaces_in_team(team_id):
+                url = f"{self.api_url}/space/{space_id}/list"
+                params = {"archived": "false"}
+                response = await self._send_api_request(url, params)
+                projects = response.get("lists")
+                yield [
+                    {**project, CustomProperties.TEAM_ID: team_id}
+                    for project in projects
+                ]
+
+    async def get_single_project(self, list_id: str) -> dict[str, Any]:
+        """Get a single project by folder_id."""
+        url = f"{self.api_url}/list/{list_id}"
+        response = await self._send_api_request(url)
+        return response
+
+    async def get_paginated_issues(self) -> AsyncGenerator[List[dict[str, Any]], None]:
+        """Get all issues in a project."""
+        if self.team_key is None:
+            await self.initialize_team_key()
+        for team_id in self.team_key:
+            url = f"{self.api_url}/team/{team_id}/task"
+            page = 0
+            while True:
+                params = {"page": page}
+                response = await self._send_api_request(url, params)
+                yield response.get("tasks")
+                if response.get("last_page"):
```

Comment:
Please share a sample response of the issues data you get from Clickup. I want to understand where you get the `last_page` key from

Reply:
```json
{
  "tasks": [
    {
      "id": "8694zyw0g",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Content Innovation",
      "text_content": "🏆 Regularly craft unique artwork that merges multiple passions or disciplines. \nBy integrating distinct disciplines or passions, you can produce groundbreaking content that not only stands out but also deeply resonates with audiences.\n\n💡 Tips\nExplore the provided Innovation tool for artistic inspiration: Innovation Tool on Instagram\nSeek to combine different passions or disciplines for fresh perspectives; for instance, merging \"cats\" and \"motorcycles\" in a creative project.\nAim for a blend that creates a captivating narrative, making your work both unique and relatable.",
      "description": "🏆 Regularly craft unique artwork that merges multiple passions or disciplines. \nBy integrating distinct disciplines or passions, you can produce groundbreaking content that not only stands out but also deeply resonates with audiences.\n\n💡 Tips\nExplore the provided Innovation tool for artistic inspiration: Innovation Tool on Instagram\nSeek to combine different passions or disciplines for fresh perspectives; for instance, merging \"cats\" and \"motorcycles\" in a creative project.\nAim for a blend that creates a captivating narrative, making your work both unique and relatable.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425788.00000000000000000000000000000000",
      "date_created": "1720029667194",
      "date_updated": "1720029667194",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718420400000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 1,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw0g",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw0e",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Content Planning",
      "text_content": "🏆 Schedule a week's worth of posts in advance, ensuring a mix of carousel images, reels, and stories. \nAn unplanned approach can lead to inconsistent posting and brand messaging. Strategically plan your content in advance for a steady stream of engaging posts.\n\n💡 Tips\nDevelop a Content Calendar to guide daily and weekly posting.\nBalance between different types of content to keep the feed fresh.\nSchedule posts during peak engagement times for maximum visibility.",
      "description": "🏆 Schedule a week's worth of posts in advance, ensuring a mix of carousel images, reels, and stories. \nAn unplanned approach can lead to inconsistent posting and brand messaging. Strategically plan your content in advance for a steady stream of engaging posts.\n\n💡 Tips\nDevelop a Content Calendar to guide daily and weekly posting.\nBalance between different types of content to keep the feed fresh.\nSchedule posts during peak engagement times for maximum visibility.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425794.00000000000000000000000000000000",
      "date_created": "1720029667050",
      "date_updated": "1720029667050",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718420400000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 1,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw0e",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw0f",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Market Research",
      "text_content": "🏆 Analyze the algorithm and stay updated on the latest trends. Think of Instagram as your market research tool. Regularly observing audience behaviors, engagement patterns, and staying informed on platform updates will ensure your content strategies are aligned with current trends. \n\n💡 Tips\nRegularly check out sources like the YouTube channel heyDominik for updates and insight.\nUtilize Instagram Insights for a data-driven understanding of audience preferences and to discern top-performing content.\nAdapt your content based on insights and feedback, ensuring it's in line with platform trends and audience expectations.",
      "description": "🏆 Analyze the algorithm and stay updated on the latest trends. Think of Instagram as your market research tool. Regularly observing audience behaviors, engagement patterns, and staying informed on platform updates will ensure your content strategies are aligned with current trends. \n\n💡 Tips\nRegularly check out sources like the YouTube channel heyDominik for updates and insight.\nUtilize Instagram Insights for a data-driven understanding of audience preferences and to discern top-performing content.\nAdapt your content based on insights and feedback, ensuring it's in line with platform trends and audience expectations.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425787.00000000000000000000000000000000",
      "date_created": "1720029667037",
      "date_updated": "1720029667037",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718679600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw0f",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw0c",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Follow Strategy",
      "text_content": "🏆 Follow niche-specific accounts and engage with their posts. \nIt's essential to be strategic about the accounts you follow, gravitating towards those that align with your brand's niche, to increase your brand's visibility and recommendation chances.\n\n💡 Tips\nFollow accounts that align with your niche to boost profile recommendations.\nPeriodically review and refine your 'following' list to ensure brand alignment.\nEngage with the posts of those you follow to foster a genuine connection.",
      "description": "🏆 Follow niche-specific accounts and engage with their posts. \nIt's essential to be strategic about the accounts you follow, gravitating towards those that align with your brand's niche, to increase your brand's visibility and recommendation chances.\n\n💡 Tips\nFollow accounts that align with your niche to boost profile recommendations.\nPeriodically review and refine your 'following' list to ensure brand alignment.\nEngage with the posts of those you follow to foster a genuine connection.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425791.00000000000000000000000000000000",
      "date_created": "1720029666936",
      "date_updated": "1720029666936",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718679600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw0c",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw09",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Content Upload",
      "text_content": "🏆 Upload 1 carousel image/reel, and 1 story to Instagram daily.\nPrioritize consistent posting on Instagram, leveraging a variety of content forms like carousel images, reels, and stories to both retain and attract followers.\n\n💡 Tips\nEmbrace the \"McDonalds\" style: produce hyper-focused and consistent content.\nUtilize different content forms, such as carousels, short reels, and stories.\nEnsure posts, especially reels, are interactive to boost engagement metrics.",
      "description": "🏆 Upload 1 carousel image/reel, and 1 story to Instagram daily.\nPrioritize consistent posting on Instagram, leveraging a variety of content forms like carousel images, reels, and stories to both retain and attract followers.\n\n💡 Tips\nEmbrace the \"McDonalds\" style: produce hyper-focused and consistent content.\nUtilize different content forms, such as carousels, short reels, and stories.\nEnsure posts, especially reels, are interactive to boost engagement metrics.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425789.00000000000000000000000000000000",
      "date_created": "1720029666540",
      "date_updated": "1720029666540",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718679600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw09",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw07",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "User Engagement",
      "text_content": "🏆 Spend 10-15 minutes interacting with the most engaged users of large, niche-aligned accounts. \nRegularly engage with potential followers, focusing on active users of influential accounts, to expand your reach and increase your profile's visibility.\n\n💡 Tips\nBegin by identifying large accounts that cater to your target audience.\nDive into their posts with the most comments, signaling high engagement.\nEngage by liking and commenting on the posts of these active commentators, and revisit the large account's post to continue this engagement cycle.",
      "description": "🏆 Spend 10-15 minutes interacting with the most engaged users of large, niche-aligned accounts. \nRegularly engage with potential followers, focusing on active users of influential accounts, to expand your reach and increase your profile's visibility.\n\n💡 Tips\nBegin by identifying large accounts that cater to your target audience.\nDive into their posts with the most comments, signaling high engagement.\nEngage by liking and commenting on the posts of these active commentators, and revisit the large account's post to continue this engagement cycle.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425793.00000000000000000000000000000000",
      "date_created": "1720029666536",
      "date_updated": "1720029666536",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718679600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw07",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw08",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Creator Messaging",
      "text_content": "🏆 Direct Message other accounts that share your target audience.\nUtilize DMs as a strategic tool to establish connections with accounts that have overlapping or desired followers. Doing so increases the chances of mutual recommendations as both accounts grow.\n\n💡 Tips\nInitiate authentic and genuine conversations.\nPrioritize DMs to accounts with overlapping follower bases.\nBe open to collaboration proposals or initiate them when deemed fit.",
      "description": "🏆 Direct Message other accounts that share your target audience.\nUtilize DMs as a strategic tool to establish connections with accounts that have overlapping or desired followers. Doing so increases the chances of mutual recommendations as both accounts grow.\n\n💡 Tips\nInitiate authentic and genuine conversations.\nPrioritize DMs to accounts with overlapping follower bases.\nBe open to collaboration proposals or initiate them when deemed fit.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425792.00000000000000000000000000000000",
      "date_created": "1720029666536",
      "date_updated": "1720029666536",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1718679600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw08",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw06",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Profile Optimization",
      "text_content": "🏆 Review and update bio with niche-specific keywords and check profile aesthetics for brand consistency. \nYour profile is the first impression users get of your brand. A well-optimized bio and cohesive aesthetic ensure a positive and clear brand message is conveyed.\n\n💡 Tips\nRegularly update the bio with niche-specific keywords.\nMaintain a consistent profile aesthetic, from images to highlight covers.\nUse a recognizable profile picture, for example a brand logo or related image.",
      "description": "🏆 Review and update bio with niche-specific keywords and check profile aesthetics for brand consistency. \nYour profile is the first impression users get of your brand. A well-optimized bio and cohesive aesthetic ensure a positive and clear brand message is conveyed.\n\n💡 Tips\nRegularly update the bio with niche-specific keywords.\nMaintain a consistent profile aesthetic, from images to highlight covers.\nUse a recognizable profile picture, for example a brand logo or related image.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425790.00000000000000000000000000000000",
      "date_created": "1720029666534",
      "date_updated": "1720029666534",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1719975600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 2,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw06",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zyw05",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Analysis & Strategy",
      "text_content": "🏆 Analyze previous month's performance metrics, then refine and adjust the content strategy accordingly. \nIn the ever-changing environment of Instagram, both analyzing past performance and refining future strategies are crucial. This combined approach ensures that your content remains relevant and effective.\n\n💡 Tips\nRely on Instagram Insights for in-depth performance data.\nIdentify and replicate strategies behind top-performing content.\nEvaluate follower growth rate and strategize ways to boost it.",
      "description": "🏆 Analyze previous month's performance metrics, then refine and adjust the content strategy accordingly. \nIn the ever-changing environment of Instagram, both analyzing past performance and refining future strategies are crucial. This combined approach ensures that your content remains relevant and effective.\n\n💡 Tips\nRely on Instagram Insights for in-depth performance data.\nIdentify and replicate strategies behind top-performing content.\nEvaluate follower growth rate and strategize ways to boost it.",
      "status": {
        "status": "to do",
        "id": "p90120581564_GW3XzuD9",
        "color": "#d3d3d3",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "119425795.00000000000000000000000000000000",
      "date_created": "1720029666529",
      "date_updated": "1720029666529",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1719975600000",
      "start_date": null,
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "020a7a56-3300-4dad-96bd-ccbb48993eae",
          "name": "Attachments",
          "type": "attachment",
          "type_config": {},
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "ea5833df-5836-48e8-ac92-e1a1f38ed378",
          "name": "Cadence",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "options": [
              {
                "id": "5b20da80-c549-4502-a186-ef6130e57dc9",
                "name": "Daily",
                "color": "#fa7e1e",
                "orderindex": 0
              },
              {
                "id": "3547fe1c-46f9-4937-b4d6-b0a2130245da",
                "name": "Weekly",
                "color": "#d62976",
                "orderindex": 1
              },
              {
                "id": "3a031009-2e76-4396-ac5e-21284587a999",
                "name": "Monthly",
                "color": "#962fbf",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720029665789",
          "hide_from_guests": false,
          "value": 2,
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zyw05",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202729089",
        "name": "90 Day Instagram Success Checklist",
        "access": true
      },
      "project": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "folder": {
        "id": "90121569135",
        "name": "hidden",
        "hidden": true,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypy7",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Homepage banner",
      "text_content": null,
      "description": null,
      "status": {
        "status": "analyzing",
        "id": "c90121568931_c16954322_c4221906_vtCck0ys",
        "color": "#5f55ee",
        "type": "custom",
        "orderindex": 3
      },
      "orderindex": "39893482.00000000000000000000000000000000",
      "date_created": "1720028606692",
      "date_updated": "1720028606692",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1709377200000",
      "start_date": "1707994800000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "13%",
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "19%",
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 3,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 1,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Bright red",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Soft pink",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypy7",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypy4",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "YouTube commercial",
      "text_content": null,
      "description": null,
      "status": {
        "status": "Open",
        "id": "c90121568931_c16954322_c4221906_QytwkQei",
        "color": "#1090e0",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "39893489.00000000000000000000000000000000",
      "date_created": "1720028606368",
      "date_updated": "1720028606368",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": {
        "color": "#6fddff",
        "id": "3",
        "orderindex": "3",
        "priority": "normal"
      },
      "due_date": "1710932400000",
      "start_date": "1709550000000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "98ae620f-8861-449b-a4b6-84ebca468da9"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "With introduction",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Without introduction",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypy4",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypy3",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Welcome to ClickUp",
      "text_content": null,
      "description": null,
      "status": {
        "status": "running",
        "id": "c90121568931_c16954322_c4221906_GQB1KirC",
        "color": "#b660e0",
        "type": "custom",
        "orderindex": 2
      },
      "orderindex": "39893487.00000000000000000000000000000000",
      "date_created": "1720028606325",
      "date_updated": "1720028606325",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1712055600000",
      "start_date": "1710414000000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "10%",
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "14%",
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 3,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "adcbc858-9e69-4897-a91d-cf6f0b1ae748"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Send to paid users",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Sent to free users",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypy3",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypy2",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Promo announcement",
      "text_content": null,
      "description": null,
      "status": {
        "status": "drafting",
        "id": "c90121568931_c16954322_c4221906_EkVVbE1l",
        "color": "#f8ae00",
        "type": "custom",
        "orderindex": 1
      },
      "orderindex": "39893484.00000000000000000000000000000000",
      "date_created": "1720028606268",
      "date_updated": "1720028606268",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": {
        "color": "#f8ae00",
        "id": "2",
        "orderindex": "2",
        "priority": "high"
      },
      "due_date": "1714734000000",
      "start_date": "1712142000000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "adcbc858-9e69-4897-a91d-cf6f0b1ae748"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Subject Line A",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Subject Line B",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypy2",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypxu",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Facebook post",
      "text_content": null,
      "description": null,
      "status": {
        "status": "Open",
        "id": "c90121568931_c16954322_c4221906_QytwkQei",
        "color": "#1090e0",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "39893488.00000000000000000000000000000000",
      "date_created": "1720028605748",
      "date_updated": "1720028605748",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": {
        "color": "#6fddff",
        "id": "3",
        "orderindex": "3",
        "priority": "normal"
      },
      "due_date": "1716030000000",
      "start_date": "1714820400000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "98ae620f-8861-449b-a4b6-84ebca468da9"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Link to checkout",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Link to product page",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypxu",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypxt",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Fall campaign",
      "text_content": null,
      "description": null,
      "status": {
        "status": "drafting",
        "id": "c90121568931_c16954322_c4221906_EkVVbE1l",
        "color": "#f8ae00",
        "type": "custom",
        "orderindex": 1
      },
      "orderindex": "39893486.00000000000000000000000000000000",
      "date_created": "1720028605746",
      "date_updated": "1720028605746",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": {
        "color": "#d8d8d8",
        "id": "4",
        "orderindex": "4",
        "priority": "low"
      },
      "due_date": "1714734000000",
      "start_date": "1713438000000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 1,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "adcbc858-9e69-4897-a91d-cf6f0b1ae748"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Header image 1",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Header image 2",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypxt",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypxr",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Adwords search term",
      "text_content": null,
      "description": null,
      "status": {
        "status": "running",
        "id": "c90121568931_c16954322_c4221906_GQB1KirC",
        "color": "#b660e0",
        "type": "custom",
        "orderindex": 2
      },
      "orderindex": "39893483.00000000000000000000000000000000",
      "date_created": "1720028605743",
      "date_updated": "1720028605743",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": {
        "color": "#f8ae00",
        "id": "2",
        "orderindex": "2",
        "priority": "high"
      },
      "due_date": "1717412400000",
      "start_date": "1712142000000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "13%",
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "15%",
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 3,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "98ae620f-8861-449b-a4b6-84ebca468da9"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Include \"CHEAP\"",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Include \"FREE\"",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypxr",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypxq",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Signup buttons",
      "text_content": null,
      "description": null,
      "status": {
        "status": "running",
        "id": "c90121568931_c16954322_c4221906_GQB1KirC",
        "color": "#b660e0",
        "type": "custom",
        "orderindex": 2
      },
      "orderindex": "39893485.00000000000000000000000000000000",
      "date_created": "1720028605733",
      "date_updated": "1720028605733",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": {
        "color": "#d8d8d8",
        "id": "4",
        "orderindex": "4",
        "priority": "low"
      },
      "due_date": "1717412400000",
      "start_date": "1714820400000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "6%",
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "13.6%",
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 3,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Try orange button\n",
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 1,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Orange button",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Green button",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypxq",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    },
    {
      "id": "8694zypxp",
      "custom_id": null,
      "custom_item_id": 0,
      "name": "Pricing table",
      "text_content": null,
      "description": null,
      "status": {
        "status": "Open",
        "id": "c90121568931_c16954322_c4221906_QytwkQei",
        "color": "#1090e0",
        "type": "open",
        "orderindex": 0
      },
      "orderindex": "39893490.00000000000000000000000000000000",
      "date_created": "1720028605729",
      "date_updated": "1720028605729",
      "date_closed": null,
      "date_done": null,
      "archived": false,
      "creator": {
        "id": 82603122,
        "username": "Iyanuoluwa Adebayo",
        "color": "",
        "email": "ioluwadunsinadebayo@gmail.com",
        "profilePicture": null
      },
      "assignees": [],
      "group_assignees": [],
      "watchers": [],
      "checklists": [],
      "tags": [],
      "parent": null,
      "priority": null,
      "due_date": "1720004400000",
      "start_date": "1717498800000",
      "points": null,
      "time_estimate": null,
      "custom_fields": [
        {
          "id": "ce3965a6-2e12-4a47-aebc-d03012db30a3",
          "name": "Conversion Rate B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "cf095e7d-7309-4222-b0b9-3e673e6fdbc6",
          "name": "Conversion rate A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "1b240392-7acf-451b-945b-87d7351f3919",
          "name": "Draft ",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "07ad2c06-9646-421c-9994-e56c80cc6add",
                "name": "Draft 1",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "fbd364b4-c99b-4a9d-8a74-a9efd2e4d7e0",
                "name": "Draft 2",
                "color": "#ff7800",
                "orderindex": 1
              },
              {
                "id": "6f6a5af3-2d42-4521-8171-53ab3c67054d",
                "name": "Draft 3",
                "color": "#bf55ec",
                "orderindex": 2
              },
              {
                "id": "de9f034c-4478-4d7b-8539-57a648f10b5a",
                "name": "Final",
                "color": "#2ecd6f",
                "orderindex": 3
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": 0,
          "required": null
        },
        {
          "id": "2cfaee63-8af8-41f2-ab1a-6222a174c93c",
          "name": "Notes",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "fd055658-1f2c-4ef9-822b-6eb469e1383b",
          "name": "Progress",
          "type": "automatic_progress",
          "type_config": {
            "tracking": {
              "subtasks": true,
              "checklists": true,
              "assigned_comments": true
            },
            "complete_on": 3
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": {
            "percent_complete": 0
          },
          "required": null
        },
        {
          "id": "a95321c2-9cd2-463a-88ae-48405f4b8e65",
          "name": "Results",
          "type": "drop_down",
          "type_config": {
            "default": 0,
            "placeholder": null,
            "new_drop_down": true,
            "options": [
              {
                "id": "ba25010b-86e4-4943-a3b4-ea7ebfd22ace",
                "name": "A",
                "color": "#81B1FF",
                "orderindex": 0
              },
              {
                "id": "b2d771a0-eac5-4cf3-a318-2357b464d9ef",
                "name": "B",
                "color": "#3397dd",
                "orderindex": 1
              },
              {
                "id": "48fcf5a2-7665-4f1d-8fec-caef235636fe",
                "name": "Same",
                "color": "#b5bcc2",
                "orderindex": 2
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "required": null
        },
        {
          "id": "a45436eb-9480-458a-b757-750197a4fafc",
          "name": "Type",
          "type": "labels",
          "type_config": {
            "options": [
              {
                "id": "98ae620f-8861-449b-a4b6-84ebca468da9",
                "label": "Advertisment",
                "color": "#e50000"
              },
              {
                "id": "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce",
                "label": "Landing Page",
                "color": "#2ecd6f"
              },
              {
                "id": "adcbc858-9e69-4897-a91d-cf6f0b1ae748",
                "label": "Email",
                "color": "#04A9F4"
              }
            ]
          },
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": [
            "c831e8f2-68a0-4ff6-87d0-0e73db7d19ce"
          ],
          "required": null
        },
        {
          "id": "73d61dc7-0370-4539-8e3a-fe6cd33e0e70",
          "name": "Version A",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Include all items",
          "required": null
        },
        {
          "id": "bfd625e1-2fa6-4c11-a0e2-15b8506ad023",
          "name": "Version B",
          "type": "text",
          "type_config": {},
          "date_created": "1720028604938",
          "hide_from_guests": false,
          "value": "Include only most important items",
          "required": null
        }
      ],
      "dependencies": [],
      "linked_tasks": [],
      "locations": [],
      "team_id": "9012068110",
      "url": "https://app.clickup.com/t/8694zypxp",
      "sharing": {
        "public": false,
        "public_share_expires_on": null,
        "public_fields": [
          "assignees",
          "priority",
          "due_date",
          "content",
          "comments",
          "attachments",
          "customFields",
          "subtasks",
          "tags",
          "checklists",
          "coverimage"
        ],
        "token": null,
        "seo_optimized": false
      },
      "permission_level": "create",
      "list": {
        "id": "901202728684",
        "name": "Tests",
        "access": true
      },
      "project": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "folder": {
        "id": "90121568931",
        "name": "New Folder",
        "hidden": false,
        "access": true
      },
      "space": {
        "id": "90120581564"
      }
    }
  ],
  "last_page": true
}

Reply:
Ahh this looks great then. Thank you